### ISSUE TITLE
Fix labels "for" attribute

### ISSUE THREAD

**Issue by ozin7** [ayrinmo/html_sandbox#6]
```
<label for="">Fornavn</label>
<input type="text">
```
you need to use the "name" attribute
```
<label for="first_name">Fornavn</label>
<input type="text" name="first_name">
```

**Comment by ozin7** [comment]
@ayrinmo all "input" elements should have the "name" attributes

**Comment by ozin7** [comment]
@ayrinmo "name" attribute should have a unique value, so if one input has name="first_name" then another fields should have another name.
Name attribute describes how the input is used, if it is set to "first_name" then we know that we should insert the first name in thiss field.
Translations:
Epostadresse -> Email
Fornavn -> First name
Etternavn -> Last name
Akademika Nyhetsbrev -> Akademika Newsletters

Please use en words.

**Comment by ayrinmo** [comment]
Yes, I realized the mistake. Today we watched a lesson on forms. I will correct. We need to see more about the label.
